src/containers/CreatePipelineRun/YAMLEditor.js

```diff
@@ -183,6 +194,7 @@ export function CreateYAMLEditor({ code: initialCode = '' }) {
             theme="dark"
             extensions={[StreamLanguage.define(codeMirrorYAML)]}
             onChange={onChange}
+            editable={!loading}
```

Comment:
We should display some form of loading state, otherwise the user won't know why the editor is disabled.

Reply:
Yes either some kind of spinner or skeleton until the resource is loaded and ready to display.

Carbon provides a number of elements for this purpose:
- https://v7-react.carbondesignsystem.com/?path=/story/components-loading--default
- https://v7-react.carbondesignsystem.com/?path=/story/components-skeleton-skeletonplaceholder--default&knob-Active%20(active)=true&knob-Classes%20with%20different%20sizes=my--skeleton__placeholder--large&knob-Description%20(description)=Active%20loading%20indicator&knob-Small%20(small)=true

Personally I think the Loading component with some text explaining what it's doing would be best here, but if you have another suggestion I'm willing to consider it.

You can find an example of how we use the Loading component in the LoadingShell while the Dashboard application is loading:
https://github.com/tektoncd/dashboard/blob/896eb37908d12c9f1d2af6d60647c5f9f32e3e83/packages/components/src/components/LoadingShell/LoadingShell.js#L81-L84